operations/system_management_health/Troubleshoot_Grafana_Dashboard.md

```diff
@@ -0,0 +1,36 @@
+# Troubleshoot Grafana Dashboard
+
+General Grafana Dashboard Troubleshooting Topics
```

Comment:
Remove unnecessary top-level link and fix capitalization in line 3. For example...

---

General Grafana dashboard troubleshooting topics:

  - [Ceph - OSD Overview Dashboard](#ceph-osd-overview-dashboard)
  - [Ceph - RBD Overview Dashboard](#ceph-rbd-overview-dashboard)
  - [Ceph - RGW Instance Detail Dashboard](#ceph-rgw-instance-detail-dashboard)
  - [Ceph - RGW Overview Dashboard](#ceph-rgw-overview-dashboard)